README.md

```diff
@@ -7,34 +7,53 @@ The code contains the [MNE](http://martinos.org/mne/) contribution to the [Bioma
 
 Results of our analysis is presented [here](http://mne-tools.github.io/mne-biomag-group-demo/)
 
-Run analysis
-------------
+Steps to replicate
+------------------
 
-To replicate our results go to the scripts/processing folder then do::
+First, clone the repository using git::
 
-   $ make check
+	$ git clone https://github.com/mne-tools/mne-biomag-group-demo.git
+
+Then go to the directory `mne-biomag-group-demo` and check out the tag `v0.1` using the command::
+
+	$ git checkout tags/v0.1
+
+This ensures that you use the same code which was used to produce the results in the paper.
+
+Next, make sure your system is properly configured. You should have the following
+installed::
+
+	Python packages: numpy, scipy, matplotlib, scikit-learn, pysurfer, mayavi, mne-python, sphinx_gallery
+	Anatomy: freesurfer
```

Comment:
you need to recommend using the conda env we should ship with this repo.

Reply:
ok sure, but it does not guarantee that things won't break because the env in MNE does not specify the version numbers. But if @larsoner wants to share his env, I can add it